Sources/Swift/Integrations/SessionReplay/SentryTouchTracker.swift

```diff
@@ -34,39 +35,48 @@ class SentryTouchTracker: NSObject {
      * will ever have the same pointer.
      */
     private var trackedTouches = [UITouch: TouchInfo]()
+    private let dispatchQueue: SentryDispatchQueueWrapper
     private var touchId = 1
     private let dateProvider: SentryCurrentDateProvider
     private let scale: CGAffineTransform
     
-    init(dateProvider: SentryCurrentDateProvider, scale: Float) {
+    init(dateProvider: SentryCurrentDateProvider, scale: Float, dispatchQueue: SentryDispatchQueueWrapper) {
         self.dateProvider = dateProvider
         self.scale = CGAffineTransform(scaleX: CGFloat(scale), y: CGFloat(scale))
+        self.dispatchQueue = dispatchQueue
+    }
+    
+    convenience init(dateProvider: SentryCurrentDateProvider, scale: Float) {
+        self.init(dateProvider: dateProvider, scale: scale, dispatchQueue: SentryDispatchQueueWrapper())
```

Comment:
```suggestion
        // SentryTouchTracker has it own dispatch queue instead of using the one
        // from Dependency container to avoid the bottleneck of sharing the same
        // queue with the rest of the SDK.
        self.init(dateProvider: dateProvider, scale: scale, dispatchQueue: SentryDispatchQueueWrapper())
```